ci/build_libtelio.py

```diff
@@ -51,6 +51,55 @@ def post_copy_libsqlite3_binary_to_dist(config, args):
         )
 
 
+def post_copy_windows_debug_symbols_to_distribution_dir(config, args):
```

Comment:
What copies debug artifacts for linux and android ?

Reply:
there's no need to copy, the debug symbols are split on the `dist/` directory

Reply:
https://github.com/NordSecurity/rust_build_utils/pull/23/files#diff-5f89a99b8914336c0a0e377629e04af79fe13086ea94ee620f03b616c368fd5eR15